Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
-ruby '2.2.2'
+ruby '2.2.3'
 
-gem 'rails', '4.1.12'
+gem 'rails', '4.1.13'
```

Comment:
Unclear question so I will ahead and tried to upgrade to Raisl 4.2.4 and did not see much except a few warning. Tried "rails s" but several things only point to production things so please check it.